public/css/index.css

```
@@ -0,0 +1,24 @@
+                  unmigrated view, exactly as before
+     paper      — reserved; first consumed in Paper Wave 1
+
+   Cache-busting query strings on @imports preserve the prior link-tag
+   versioning behavior. Bump them when shipping a stylesheet update.
+
+   See docs/superpowers/specs/2026-05-09-paper-tokens-cascade-layers-refactor-design.md
+   ════════════════════════════════════════════════════════════════════ */
+
+@layer components, legacy, paper;
+
+@import url('../styles.css?v=86')      layer(components);
```

Stylelint (17.10.0)

[error] Expected "url('../styles.css?v=86')" to be "'../styles.css?v=86'" (import-notation)

(import-notation)